src/containers/pages/JurisdictionAssignment/JurisdictionTable/index.tsx

```diff
@@ -351,11 +279,12 @@ const JurisdictionTable = (props: JurisdictionSelectorTableProps) => {
               onChange={onParentCheckboxClick}
             />
           </th>
-          <th style={{ width: '70%' }}>{headerItems[1]}</th>
+          <th>{headerItems[1]}</th>
           <th>{headerItems[2]}</th>
           <th>{headerItems[3]}</th>
           <th>{headerItems[4]}</th>
           <th>{headerItems[5]}</th>
```

Comment:
I prefer having `headerItems` as a map/object instead of resorting to magic numbers like this.